running-adapter.md

```diff
@@ -0,0 +1,55 @@
+# Running the REST FileSystem Adapter
+
+## Using Docker
+
+### Using Dropwizard Docker Image
+
+> :warning: Docker cannot be installed on Virtual Machines in our infrastructure.
+
+Make sure [Docker for Desktop](https://docs.docker.com/desktop/install/windows-install/) is installed.
```

Comment:
This requires a license if running within the company.  We generally use an alternative like Rancher Desktop instead.